labgrid/driver/sshdriver.py

```diff
@@ -60,6 +61,19 @@ def _start_own_master(self):
 
         timeout = 30
 
+        # Retry start of controlmaster, to allow handle failures such as
+        # connection refused during target startup
+        retry_deadline = time.time() + timeout
+        while True:
+            try:
+                return self._start_own_master_once(timeout)
+            except ExecutionError as e:
```

Comment:
This should include a short sleep (maybe 0.5 seconds) to avoid degrading into a busy poll.

Reply:
Done.